src/core/link-to-dfn.js

```diff
@@ -132,14 +136,8 @@ function collectDfns(title) {
  * @param {HTMLAnchorElement} anchor
  * @param {CaseInsensitiveMap} titleToDfns
  */
-function findLinkTarget(target, anchor, titleToDfns) {
+function useLinkTarget(target, anchor, titleToDfns) {
```

Comment:
`mutateLinkTarget` if its sole purpose is mutation?

Reply:
It also mutates `possibleExternalLinks`

Reply:
AFAICT this function doesn't touch `possibleExternalLinks`, though?

Reply:
Ah okay, GitHub skipped some lines. Well, in that case, maybe `process`? It's also way too general but better than `use` as the latter doesn't imply it'll be mutated.

Reply:
Or maybe refactor this function further to just return the anchor is a possible external link or not.

Reply:
This refactor is turning out to be more difficult then I expected..

Reply:
Maybe just define a local boolean variable and return it at the last line?

Reply:
I was trying to:
```js
if (hasLocalTarget(anchor, dfn, titleToDfns)) {
  processAnchor(anchor, dfn);
} else {
  possibleExternalLinks.push(anchor);
}
```

Didn't work out fine. Went with your suggestion in https://github.com/w3c/respec/pull/2836/commits/505c12eee8aeeb59a180c08e5576d71362cf516f.

Reply:
That's a great refactoring if possible, sad that it's harder than expected 😅

Somehow I prefer `mutate` for the name as now that's the main purpose of the function.

Reply:
This turned out to be difficult as the order of conditions mattered here. We would be essentially repeating all conditionals to check if it has local target, then use same conditionals to mutate the anchor.
Is the present one acceptable?

Reply:
>Is the present one acceptable?

Yup 👍